src/libraries/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/ManagedTextSection.cs

```diff
@@ -90,7 +90,7 @@ internal sealed class ManagedTextSection
         /// If set, the module contains instructions that assume a 64 bit instruction set. For example it may depend on an address being 64 bits.
         /// This may be true even if the module contains only IL instructions because of PlatformInvoke and COM interop.
         /// </summary>
-        internal bool Requires64bits => Machine == Machine.Amd64 || Machine == Machine.IA64 || Machine == Machine.Arm64;
+        internal bool Requires64bits => Machine == Machine.Amd64 || Machine == Machine.IA64 || Machine == Machine.Arm64 || Machine == Machine.Riscv64;
```

Comment:
This is only needed if Windows got ported to Riscv64. 

There are many 64-bit architectures missing in this check and it is not causing any issues.

I would keep this simple and omit the Riscv check from here.

Reply:
Thanks!